projects/frontend/data-pipelines/gui/e2e/integration/lib/explore/data-jobs.spec.js

```diff
@@ -122,25 +124,25 @@ describe(
             );
         });
 
-        it("Data Jobs Explore Page - searches data jobs", () => {
-            cy.log("Fixture for name: " + testJobs[0].job_name);
-
-            dataJobsExplorePage = DataJobsExplorePage.navigateTo();
-
-            dataJobsExplorePage
-                .getDataGridCell(testJobs[1].job_name)
-                .should("be.visible");
-
-            dataJobsExplorePage.searchByJobName(testJobs[0].job_name);
-
-            dataJobsExplorePage
-                .getDataGridCell(testJobs[0].job_name)
-                .should("be.visible");
-
-            dataJobsExplorePage
-                .getDataGridCell(testJobs[1].job_name)
-                .should("not.exist");
-        });
+        // it("Data Jobs Explore Page - searches data jobs", () => {
```

Comment:
Can you please add a TODO with the ticket link to these tests?

Reply:
I've linked the issue to the PR description instead, 
I prefer keeping the code comments lean -I've seen multiple occurrences of issue-referring code comment leftovers.